neo/rawio/spikeglxrawio.py

```diff
@@ -140,15 +141,20 @@ def _parse_header(self):
             self._memmaps[key] = data
 
         # create channel header
+        signal_buffers = []
         signal_streams = []
         signal_channels = []
         for stream_name in stream_names:
             # take first segment
             info = self.signals_info_dict[0, stream_name]
 
+            buffer_id = stream_name
+            buffer_name = stream_name
+            signal_buffers.append((buffer_name, buffer_id))
```

Comment:
At the moment the mapping buffer to stream is one to one.
But after this PR we will be able to split buffer into logical stream super easily.